### PR TITLE
Metrics queries

### PR DESCRIPTION
**Pull request by alxswny**
Created two queries: one subsets pbf data and configures it for use within the other query which gathers metrics from each pbf time slice.

### CODE REVIEW

File: queries/subset_data_for_mterics.sql [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+    wetland,
+    all_tags,
+    '2019'::int as year,
+    ST_GeometryType(wkb_geometry) as geom_type,
+    CASE WHEN ST_GeometryType(wkb_geometry) IN ('ST_LineString') 
+         THEN ST_Length(wkb_geometry::geography) ELSE NULL END as length,
+    CASE WHEN ST_GeometryType(wkb_geometry) IN ('ST_MultiPolygon')
+         THEN ST_Area(wkb_geometry::geography) ELSE NULL END as area
+FROM mn_2019_upload.multipolygons
+;
```
Comment (mattmanley) [inline queries/subset_data_for_mterics.sql]: Should add one line at the end of this file

File: queries/subset_data_for_mterics.sql [hunk 2]
```diff
@@ -0,0 +1,158 @@
+/*
+These queries create tables from 2009, 2011, and 2019 linear and multipolygon data as a base for a larger metrics query.
```
Comment (mattmanley) [inline queries/subset_data_for_mterics.sql]: Break this comment onto two lines for readability

**Review by mattmanley — CHANGES_REQUESTED** [review]
LGTM aside from two small formatting things.

File: queries/subset_data_for_mterics.sql (hunk 2 above)
Comment (alxswny) [reply queries/subset_data_for_mterics.sql]: @mattmanley addressed both of these comments.

**Review by mattmanley — APPROVED** [review]

**Pull request merged**